tests/storage/cbt/utils.py

```
@@ -29,22 +30,100 @@ def cbt_pvc_size_with_headroom(source_disk_size: str, headroom_gib: int = 10) ->
+    """
+    LOGGER.info(f"Waiting for guest device of volume {volume_name} on VM {vm.name}")
+    for target in TimeoutSampler(
+        wait_timeout=TIMEOUT_10MIN,
+        sleep=TIMEOUT_5SEC,
+        func=guest_volume_target,
+        vm=vm,
+        volume_name=volume_name,
+    ):
+        if target:
+            return f"/dev/{target}"
+    raise TimeoutExpiredError(f"Volume {volume_name} on VM {vm.name} never reported a guest device name")
```

Ruff (0.16.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)